OpenXmlPowerTools/Word/DocumentAssembler.cs

```diff
@@ -514,7 +514,8 @@ private static object TransformToMetadata(XNode node, TemplateError te)
                     OpenXmlRegex.Replace(new[] { element }, r, thisGuid, (_, match) =>
                     {
                         var matchString = match.Value.Trim();
-                        var xmlText = matchString.Substring(2, matchString.Length - 4).Trim().Replace('“', '"').Replace('”', '"');
+                        var xmlText = CleanInvalidXmlChars(matchString.Substring(2, matchString.Length - 4).Trim().Replace('“', '"').Replace('”', '"'));
```

Comment:
@luizfbicalho I am not sure that DocumentAssembler actually should replace invalid XML characters.

Imagine for a moment that `DocumentAssemblier` (sort of interpretator) , that parse `*.docx` (kind of source code)  and run instructions against XML (data). What do you expect from the interpretator when you send an invalid source code for execution? An error that the source program is incorrect with position details or some magic attempt inside the code to auto-correct the source and potentially invalid/unexpected execution result?

I believe that when sourcing (`*.docx`) contains invalid characters inside XML block, we should report an error with instructions on how to find it.

Reply:
The problem is not in what you send, the problem is in what word puts, look at this case that is the example, the unicode invalid char isn't in the word document, it's probably some line break or page break that turns into this invalid char

I understand that if I put something wrong the assembler should not correct, but the word format breaks the xml and this is what I'm correcting